homeassistant/components/dsmr/config_flow.py

```diff
@@ -208,6 +226,7 @@ async def async_step_setup_network(
                 vol.Required(CONF_HOST): str,
                 vol.Required(CONF_PORT): int,
                 vol.Required(CONF_DSMR_VERSION): vol.In(DSMR_VERSIONS),
+                vol.Optional(CONF_IS_RFXTRX): bool,
```

Comment:
This should be `vol.Required`, it will always be in `user_input` (and you assume that also below).

Reply:
Changed as requested.